### PR TITLE
bump dependencies

### PR DESCRIPTION
**Pull request by ph4r05**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -26,14 +26,14 @@ require (
...
-	github.com/nirasan/go-oauth-pkce-code-verifier v0.0.0-20170819232839-0fbfe93532da
+	github.com/nirasan/go-oauth-pkce-code-verifier v0.0.0-20220510032225-4f9f17eaec4c
 	github.com/pquerna/otp v1.4.0
 	github.com/prometheus/client_golang v1.19.1
 	github.com/tstranex/u2f v1.0.0
 	github.com/vjeantet/ldapserver v1.0.1
 	golang.org/x/crypto v0.24.0
 	golang.org/x/net v0.26.0
-	golang.org/x/oauth2 v0.21.0
+	golang.org/x/oauth2 v0.19.0
```
Comment (cviecco) [inline go.mod:36]: Downgrades many places maybve not fully connected when doing upgrades? As it will need to reject.

File: go.mod [hunk 2]
```diff
@@ -43,13 +43,13 @@ require (
...
 	github.com/ProtonMail/go-crypto v1.0.0 // indirect
 	github.com/aws/aws-sdk-go-v2/service/internal/accept-encoding v1.11.2 // indirect
-	github.com/aws/aws-sdk-go-v2/service/secretsmanager v1.30.0 // indirect
-	github.com/aws/aws-sdk-go-v2/service/ssooidc v1.24.5 // indirect
-	github.com/cloudflare/circl v1.3.9 // indirect
-	github.com/cyphar/filepath-securejoin v0.2.5 // indirect
+	github.com/aws/aws-sdk-go-v2/service/secretsmanager v1.28.6 // indirect
+	github.com/aws/aws-sdk-go-v2/service/ssooidc v1.26.0 // indirect
+	github.com/cloudflare/circl v1.3.7 // indirect
+	github.com/cyphar/filepath-securejoin v0.2.4 // indirect
```
Comment (cviecco) [inline go.mod:52]: Downgrades

File: go.mod [hunk 3]
```diff
@@ -82,18 +83,20 @@ require (
...
+	github.com/google/certificate-transparency-go v1.1.8 // indirect
 	github.com/google/uuid v1.6.0 // indirect
 	github.com/jmespath/go-jmespath v0.4.0 // indirect
 	github.com/lor00x/goldap v0.0.0-20180618054307-a546dffdd1a3 // indirect
 	github.com/mitchellh/mapstructure v1.5.0 // indirect
 	github.com/prometheus/client_model v0.6.1 // indirect
-	github.com/prometheus/common v0.54.0 // indirect
-	github.com/prometheus/procfs v0.15.1 // indirect
+	github.com/prometheus/common v0.52.3 // indirect
+	github.com/prometheus/procfs v0.13.0 // indirect
```
Comment (cviecco) [inline go.mod:93]: downgrades

File: go.mod [hunk 4]
```diff
@@ -82,18 +83,20 @@ require (
...
 	github.com/x448/float16 v0.8.4 // indirect
+	golang.org/x/mod v0.17.0 // indirect
 	golang.org/x/sys v0.21.0 // indirect
 	golang.org/x/time v0.5.0
-	google.golang.org/protobuf v1.34.2 // indirect
+	golang.org/x/tools v0.20.0 // indirect
+	google.golang.org/protobuf v1.33.0 // indirect
```
Comment (cviecco) [inline go.mod:99]: downgrade

**Review by cviecco — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (ph4r05) [reply go.mod]: fixed

File: go.mod (hunk 2 above)
Comment (ph4r05) [reply go.mod]: fixed

File: go.mod (hunk 3 above)
Comment (ph4r05) [reply go.mod]: fixed

File: go.mod (hunk 4 above)
Comment (ph4r05) [reply go.mod]: fixed

**Comment by ph4r05** [comment]
pls re-review

**Review by cviecco — APPROVED** [review]

**Pull request merged**